backend/coins/btc/blockchain/mocks/mock.go

```diff
@@ -0,0 +1,112 @@
+package mocks
+
+import (
+	chainhash "github.com/btcsuite/btcd/chaincfg/chainhash"
+	wire "github.com/btcsuite/btcd/wire"
+	btcutil "github.com/btcsuite/btcutil"
+	blockchain "github.com/digitalbitbox/bitbox-wallet-app/backend/coins/btc/blockchain"
+)
+
+// BlockchainMock implements Interface for use in tests.
```

Comment:
I'd say "implements blockchain.Interface". It was clear which interface as long as this BlockchainMock struct was in the same blockchain package, but it's split into "mocks" now.

Reply:
correct, done